src/utils/BatchManager.js

```diff
@@ -1,5 +1,8 @@
-function errorToSerializable(err) {
-  if (!err) return null;
+function errorToSerializable(error) {
+  if (error === undefined) return null;
```

Comment:
Yeah we're calling this for every job regardless if there's an error or not.

I agree it's a code smell, I'll change it at call-site. Although I wonder if we should still have this guard in here just in case we call it with no error.